README.adoc

```diff
@@ -277,6 +278,25 @@ Roboto Mono:
     - "/Users/user/.fontist/fonts/RobotoMono-VariableFont_wght.ttf"
 ----
 
+=== Work with Fontconfig: `fontist fontconfig`
```

Comment:
@alexeymorozov can you help add more information about what fontconfig is, what platform it runs on, and how it is useful? Thanks.

Reply:
Updated, should be better now.